src/huggingface_hub/hf_file_system.py

```diff
@@ -882,20 +1138,3 @@ def _raise_file_not_found(path: str, err: Optional[Exception]) -> NoReturn:
 
 def reopen(fs: HfFileSystem, path: str, mode: str, block_size: int, cache_type: str):
     return fs.open(path, mode=mode, block_size=block_size, cache_type=cache_type)
-
-
-# Add docstrings to the methods of HfFileSystem from fsspec.AbstractFileSystem
-for name, function in inspect.getmembers(HfFileSystem, predicate=inspect.isfunction):
-    parent = getattr(fsspec.AbstractFileSystem, name, None)
```

Comment:
I still think a similar logic makes sense for all "hidenn" methods not defined in huggingface_hub but in fsspec instead: read_text, read-bytes, write_text, write_bytes, tail, touch, etc. The initial goal was to make them appear in the documentation (see https://github.com/huggingface/huggingface_hub/issues/2100) even though the implementation did not do that.

This can be done in a later PR (not urgent)